src/file/watcher.go

```diff
@@ -40,6 +40,7 @@ var (
 type Event struct {
 	Op       Op
 	Path     string
+	OldChecksum string
```

Comment:
Maybe `ServerChecksum` or `RemoteChecksum`?

Reply:
I think `PreviousChecksum` might be better because the previous checksum might come from a lockfile in the future.